TestProjects/UniversalGraphicsTest_Lighting/Assets/Scenes/031_Shader_GlossyEnvironmentSky.unity

```diff
@@ -76,21 +77,30 @@ LightmapSettings:
     m_PVRDirectSampleCount: 32
     m_PVRSampleCount: 500
     m_PVRBounces: 2
+    m_PVREnvironmentSampleCount: 500
+    m_PVREnvironmentReferencePointCount: 2048
+    m_PVRFilteringMode: 2
+    m_PVRDenoiserTypeDirect: 0
+    m_PVRDenoiserTypeIndirect: 0
+    m_PVRDenoiserTypeAO: 0
     m_PVRFilterTypeDirect: 0
     m_PVRFilterTypeIndirect: 0
     m_PVRFilterTypeAO: 0
-    m_PVRFilteringMode: 1
```

Comment:
Not sure what this settings changes to, but we should disable filtering for our GI tests. We don't want to test filtering/denoiser in our tests and by having it enabled it just add the risk that our test fails when you make changes to filtering.

Reply:
m_PVRFilteringMode is an enum describing None, Auto, Advanced for filtering and denoising.
The value isn't used if there are no lightmaps in the scene (there are no lightmaps in this case so it doesn't change anything).